localtools/src/integrationTest/kotlin/testconfiguration/util/AppResources.kt

```diff
@@ -8,7 +8,7 @@ object AppResources {
     // and libs are compatible with.  Local testing for unpublished versions of the smart contract's WASM file can be
     // done by tweaking ManagedProvenanceTestContainer's afterStartup function implementation to fetch the file from local
     // resources, if needed.
-    const val CONTRACT_VERSION = "3.2.0"
+    const val CONTRACT_VERSION = "3.3.0"
```

Comment:
Latest contract version that supports v1.14.0's new MsgFees change that no longer removes 50% of amounts sent via the `asset_custom_fee` msg in contracts